pkgs/by-name/po/porn-vault/package.nix

```diff
@@ -0,0 +1,112 @@
+{
+  fetchFromGitLab,
+  fetchurl,
+  lib,
+  pnpm,
+  stdenvNoCC,
+  nodejs_22,
+  ffmpeg,
+  imagemagick,
+  writeShellScript,
+  makeWrapper,
+  autoPatchelfHook,
+}:
+let
+  izzy = stdenvNoCC.mkDerivation rec {
+    pname = "izzy";
+    version = "2.0.2";
+
+    src = fetchurl {
+      url = "https://gitlab.com/porn-vault/izzy/-/package_files/151366122/download";
+      hash = "sha256-K+rVb2F3HizMKIpY1zbLj5zE7B6DkIoq8nJYPUpQn+Y=";
+    };
+
+    dontUnpack = true;
+
+    nativeBuildInputs = [
+      autoPatchelfHook
+    ];
+
+    installPhase = ''
```

Comment:
Nit: missing `runHook`s for	`{pre,post}Install`

Reply:
I was able to compile izzy from source, so this will be gone soon (will force push later today, still resolving other comments)